File: Hayaa.ServiceFrame/ServiceFrameClient/Hayaa.ServicePlatform.Client/Config/ConfigHelper.cs
using Hayaa.ConfigSeed.Standard;


namespace Hayaa.ServicePlatform.Client.Config
{
    internal class ConfigHelper: ConfigTool<ServicePlatformServiceConfig, ServicePlatformServiceRootConfig>
    {
        private static ConfigHelper _instance = new ConfigHelper();
        private ConfigHelper() : base(DefineTable.ServicePlatformClientComponetID)
        {

        }

        internal static ConfigHelper Instance { get => _instance; }
    }
}

File: Hayaa.ServiceFrame/ServiceFrameClient/hayaa.serviceplatform.client_java/hayaa.serviceplatform.client/src/main/java/hayaa/serviceplatform/client/ConfigHelper.java
package hayaa.serviceplatform.client;

import Hayaa.ConfigSeed.ConfigTool;

class ConfigHelper extends ConfigTool<ServicePlatformServiceConfig, ServicePlatformServiceRootConfig> {
    private static ConfigHelper _instance = new ConfigHelper(DefineTable.ServicePlatformClientComponetID
            ,ServicePlatformServiceConfig.class,ServicePlatformServiceRootConfig.class);
    private ConfigHelper(int componentID, Class<ServicePlatformServiceConfig> componentConfigType,
                        Class<ServicePlatformServiceRootConfig> appRootConfigType) {
        super(componentID, componentConfigType, appRootConfigType);
    }
    public static ConfigHelper get_instance() {
        return _instance;
    }

}
